parmed/amber/_amberparm.py

```diff
@@ -379,7 +380,8 @@ def from_structure(cls, struct, copy=False):
                 dt.per = 1.0
         inst.remake_parm()
         inst._set_nonbonded_tables(nbfixes)
-        if inst.pointers['NCOPY'] >= 2:
+        n_copy = inst.pointers.get('NCOPY', 1)
+        if n_copy >= 2:
```

Comment:
haha, yes.  Thanks :smiley:

I already merged this, though.  As I said, it looks good (and the coverage appears higher the way you have it)